bigbluebutton-html5/imports/api/voice-users/server/methods.js

```diff
@@ -2,12 +2,16 @@ import { Meteor } from 'meteor/meteor';
 import mapToAcl from '/imports/startup/mapToAcl';
 import listenOnlyToggle from './methods/listenOnlyToggle';
 import muteToggle from './methods/muteToggle';
+import muteAllToggle from './methods/muteAllToggle';
+import muteAllExceptPresenterToggle from './methods/muteAllExceptPresenterToggle';
 import ejectUserFromVoice from './methods/ejectUserFromVoice';
 
 Meteor.methods(mapToAcl(['methods.listenOnlyToggle', 'methods.toggleSelfVoice',
-  'methods.toggleVoice', 'methods.ejectUserFromVoice'], {
+  'methods.toggleVoice', 'methods.ejectUserFromVoice', 'methods.muteAllUsers', 'methods.muteAllExceptPresenter'], {
```

Comment:
we removed ACL in https://github.com/bigbluebutton/bigbluebutton/pull/6226 - check it to see how to adjust pls